.env

```diff
@@ -33,7 +33,7 @@ ESPRESSO_SEQUENCER4_API_PORT=24004
 ESPRESSO_SEQUENCER_URL=http://sequencer0:${ESPRESSO_SEQUENCER_API_PORT}
 ESPRESSO_SEQUENCER_MAX_CONNECTIONS=25
 ESPRESSO_SEQUENCER_STORAGE_PATH=/store/sequencer
-ESPRESSO_SEQUENCER_GENESIS_FILE=/genesis/demo.toml
+ESPRESSO_SEQUENCER_GENESIS_FILE=data/genesis/demo.toml
```

Comment:
I think this will break the docker demo where the genesis files are at `/genesis`

https://github.com/EspressoSystems/espresso-sequencer/blob/15c937cea4335750929dbb57839513ef562af259/docker/sequencer.Dockerfile#L32